### ISSUE TITLE
jca release 

### ISSUE THREAD

**Issue by shroffk** [epics-base/jca#17]
`mvn release:prepare `
fails to appropriately commit the updated pom before committing it.

```
[INFO] Checking in modified POMs...
[INFO] Executing: cmd.exe /X /C "git add -- pom.xml"
[INFO] Working directory: C:\git\jca
[INFO] Executing: cmd.exe /X /C "git status"
[INFO] Working directory: C:\git\jca
[INFO] Tagging release with the label jca-2.3.7...
[INFO] Executing: cmd.exe /X /C "git tag -F C:\cygwin64\tmp\maven-scm-1402791373.commit jca-2.3.7"
[INFO] Working directory: C:\git\jca
[INFO] Executing: cmd.exe /X /C "git push https://*****@github.com/epics-base/jca.git jca-2.3.7"
[INFO] Working directory: C:\git\jca
[INFO] Executing: cmd.exe /X /C "git ls-files"
[INFO] Working directory: C:\git\jca
[INFO] Transforming 'Java Channel Access'...
[INFO] Not removing release POMs
[INFO] Checking in modified POMs...
[INFO] Executing: cmd.exe /X /C "git add -- pom.xml"
[INFO] Working directory: C:\git\jca
[INFO] Executing: cmd.exe /X /C "git status"
[INFO] Working directory: C:\git\jca
[INFO] Release preparation complete.
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
```

The pom with the non snapshot version gets added to the  but not committed before the tag is added and pushed to github

**Comment by shroffk** [comment]
@ralphlange @berryma4 any ideas?

**Comment by ralphlange** [comment]
Hmm...
Given the code reorganization, removal of the jni implementation and merge of the caj module:
Is stepping up the bugfix number the appropriate version step?

**Comment by shroffk** [comment]
Good point.
However the API is still the same with no changes.
should this be 2.4.0?

**Comment by ralphlange** [comment]
I would feel more comfortable with 2.4.0, yes.
(Mainly removal of the JNI parts is something that is significant enough to move the minor number.)

**Comment by ralphlange** [comment]
Issue has been resolved with release of jca 2.4.0. Closing.
